static/app/views/automations/new.tsx

```diff
@@ -24,7 +39,15 @@ export default function AutomationNew() {
             {t('Next')}
           </LinkButton>
         </Flex>
-      </StickyFooter>
+      </StyledStickyFooter>
     </NewAutomationLayout>
   );
 }
+
+const ContentWrapper = styled('div')`
+  position: relative;
+`;
+
+const StyledStickyFooter = styled(StickyFooter)`
+  z-index: ${p => p.theme.zIndex.footer};
```

Comment:
Should be able to use the default here.

```suggestion
  z-index: ${p => p.theme.zIndex.initial};
```

Reply:
fixed https://github.com/getsentry/sentry/pull/90008/commits/98240c136c87fb75947fd21cd60036d0ea7ddf2a